user/deployment/google-app-engine.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Google App Engine Deployment
+layout: en
+permalink: /user/deployment/google-app-engine/
+---
+
+Travis CI can automatically deploy your [Google App Engine](https://cloud.google.com/appengine/docs) or [Managed VMs](https://cloud.google.com/appengine/docs/managed-vms/) application after a successful build.
+
+For a minimal configuration, all you need to do is add the following to your `.travis.yml`:
+
+    deploy:
+      provider: gae
+      keyfile: "YOUR SERVICE ACCOUNT JSON FILE"
+      default: true
+      project: "YOUR PROJECT ID"
+
+You can create a Service Account by going to the [Google Cloud Console](console.developers.google.com), go to "APIs & auth" -> "Credentials",
+then click "Add Credential" and "Service Account", finally clicking "JSON" to downloa dthe JSON key.
+
+You must then encrypt your key before committing it to a repo. First make sure you have the Travis command line tool installed.
```

Comment:
It is technically incorrect that you "must". But it is a *_very_* good idea.

Please add a link to http://docs.travis-ci.com/user/encrypting-files/, too.